### ISSUE TITLE
xorg.conf disappears and coolbits is disabled

### ISSUE THREAD

**Issue by civyshk** [civyshk/nvidiafancurve#2]
/etc/X11/xorg.conf allows me to enable coolbits. It works but it is periodically removed (by a nvidia service?), specifically it was renamed to xorg.DDMMYYYY, so it wouldn't load at startup and the script suddenly stopped working. As a fix, I quitted this approach and created a 51-nvidia-coolbits.conf in /usr/share/X11/xorg.conf.d/ with the following content:

    Section "Device"
        Identifier     "Device0"
        Driver         "nvidia"
        VendorName     "NVIDIA Corporation"    
        Option         "Coolbits" "5"
    EndSection
    
    #blank end line

**Comment by nan0s7** [comment]
Yeah it seems to be whenever you update the drivers it resets the xorg.conf file. You can place that same file you made, in /etc/X11/xorg.conf.d/ and it can be named whatever you like. If you (or anyone viewing this) needs information on how to do this, I have put a section in my USAGE.md file in my repository with instructions on how to do this for any computer.
